src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,6 @@
 package mate.academy.service;
 
-public class AuthenticationService {
+public class AuthenticationService extends UserService {
```

Comment:
No need to extend UserService here
```suggestion
public class AuthenticationService {
```